subprojects/core/src/main/java/org/gradle/process/internal/JavaForkOptionsInternal.java

```diff
@@ -25,4 +25,14 @@ public interface JavaForkOptionsInternal extends JavaForkOptions {
      */
     boolean isCompatibleWith(JavaForkOptions options);
 
+    /**
+     * Sets extra JVM arguments to a Java process without checking debug configuration.
+     * */
+    void installJvmArgs(Iterable<?> jvmArgs);
```

Comment:
The `install` terminology seems odd here. Why not something like `replace`?

Reply:
@jvandort Sounds good to me, I'll change it